unirecfilter/README.md

```diff
@@ -64,15 +66,33 @@ Almost all data types from unirec are supported:
 
 ### Format
 #### Command line
-Filter specified on command line with `-F` flag is a single expression which is evaluated for the output interface.
+Filter specified on command line with `-F` flag is a single expression which is evaluated for the output interface. For example: `-F "SRC_PORT == 23"`
 
 #### File
-Filter specified in a file provides more flexibility. Format of the file is `[TEMPLATE_1]:FILTER_1;...;[TEMPLATE_N]:FILTER_N;` where each semicolon separated item corresponds with one output interface. One-line comments starting with `#` are allowed. To reload filter while unirecfilter is running, send signal SIGUSR1 (10) to the process.
+Filter specified in a file provides more flexibility. Format of the file is `TEMPLATE_1:FILTER_1;...;TEMPLATE_N:FILTER_N;` where each semicolon separated item corresponds with one output interface. One-line comments starting with `#` are allowed. To reload filter while unirecfilter is running, send signal SIGUSR1 (10) to the process.
 
 ## Default values
-You can use syntax FIELD=value in the template to specify default value used if field is not present on the input (f.e. uint32 BAR=1)
+You can use syntax FIELD=value in the template to specify default value used if field is not present on the input (f.e. uint32 BAR=1). Example bellow.
+`./unirecfilter -i u:test_ifc_in,u:test_ifc_out -O "ipaddr SRC_IP, ipaddr DST_IP, uint16 SRC_PORT, uint16 DST_PORT, string MESSAGE=\"this_is_message\""`
```

Comment:
Are you sure it is possible to put spaces into -O?

Reply:
Yeah, I have tested each command I have added to this README.